watchman/__main__.py

```diff
@@ -0,0 +1,21 @@
+# (c) Nelen & Schuurmans.  GPL licensed, see LICENSE.rst.
+# -*- coding: utf-8 -*-
+
+import logging.config
+import sys
+
+from . import config as cfg
+
+# Configure logging before importing get_notifier.
+logging.config.dictConfig(cfg.LOGGING)
+
+
+def main():
+    from .notify import get_notifier
+
+    notifier = get_notifier()
+    notifier.loop()
+
+
+if __name__ == "__main__":
+    sys.exit(main())
```

Comment:
Waarom deze file? Wil je ergens `python watchman` kunnen doen? Ik wist niet eens dat het kon!

Reply:
Dit maakt het mogelijk om `/venv/bin/python -m watchman` te doen. Verder kan ik zo logging configureren voordat `logger = logging.getLogger(__name__)` wordt gedaan in `notify.py`. Voorheen werd het andersom gedaan en daardoor gedroeg logging zich een beetje vreemd.

Reply:
Je nu kan zelfs `python watchman` doen, dus zonder de `-m`.

Reply:
Zonder `-m` krijg ik dit (ben er verder niet ingedoken):
```
root@fa31edf2ed59:/code# /venv/bin/python watchman
Traceback (most recent call last):
  File "/usr/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/usr/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "watchman/__main__.py", line 7, in <module>
    from . import config as cfg
ImportError: attempted relative import with no known parent package
```